### PR TITLE
AG-5105 - Add enabled property to `AgCrossLineOptions` and `AgCrossLi…

### PR DESCRIPTION
**Pull request by manapeirov**
- Add `enabled` flag to crossLine options for consistency
- Change rendering order depending on type of crossLine

### CODE REVIEW

File: charts-packages/ag-charts-community/src/chart/crossline/crossLine.ts [hunk 1]
```diff
@@ -77,7 +79,7 @@ export class CrossLine {
     }
 
     update(visible: boolean) {
-        if (!this.type) { return; }
+        if (!this.enabled || !this.type) { return; }
```
Comment (alantreadway) [inline charts-packages/ag-charts-community/src/chart/crossline/crossLine.ts:82]: Does the update case work correctly? I.e. for an existing chart with crossLines enabled, does disabling them actually remove them?

File: charts-packages/ag-charts-community/src/chart/crossline/crossLine.ts [hunk 2]
```diff
@@ -96,8 +98,10 @@ export class CrossLine {
             this.updateRangeNode();
         }
 
-        this.updateLabel();
-        this.positionLabel();
+        if (this.label.enabled) {
```
Comment (alantreadway) [inline charts-packages/ag-charts-community/src/chart/crossline/crossLine.ts:101]: Ditto

File: charts-packages/ag-charts-community/src/chart/crossline/crossLine.ts (hunk 1 above)
Comment (manapeirov) [reply charts-packages/ag-charts-community/src/chart/crossline/crossLine.ts:82]: yes

**Comment by manapeirov** [comment]
@alantreadway PTAL

**Pull request merged**